src/bucket.ts

```diff
@@ -3174,6 +3185,201 @@ class Bucket extends ServiceObject {
     }
   }
 
+  uploadDirectory(
+    directoryPathString: string,
+    options?: UploadDirectoryOptions
+  ): Promise<UploadDirectoryResponse>;
+  uploadDirectory(
+    directoryPathString: string,
+    options: UploadDirectoryOptions,
+    callback: UploadDirectoryCallback
+  ): void;
+  uploadDirectory(
+    directoryPathString: string,
+    callback: UploadDirectoryCallback
+  ): void;
+  /**
+   * @typedef {object} UploadDirectoryOptions Configuration options for Bucket#uploadDirectory().
+   * @param {boolean} [options.recurse] Whether to recursevely upload all files from
```

Comment:
I *think* these should be `@property` tags

Reply:
Also looks like a typo here

Reply:
Done, thanks